/**:
  ros__parameters:
    use_operation_mode_availability: true
    rate: 10.0
    input_qos_depth: 1000
    graph_qos_depth: 1

    use_command_mode_mappings: true
    command_mode_mappings:
      - "{mode: 1001, path: /autoware/modes/stop}"
      - "{mode: 1002, path: /autoware/modes/autonomous}"
      - "{mode: 1003, path: /autoware/modes/local}"
      - "{mode: 1004, path: /autoware/modes/remote}"
      - "{mode: 2001, path: /autoware/modes/emergency_stop}"
      - "{mode: 2002, path: /autoware/modes/comfortable_stop}"
      - "{mode: 2003, path: /autoware/modes/pull_over}"
